packages/dapp/src/components/Listings.tsx

```diff
@@ -14,208 +13,100 @@ const StyledDiv = styled.div`
   color: black;
 `;
 
-export interface ListingsState {
-  applications: List<ListingWrapper>;
-  whitelistedListings: List<ListingWrapper>;
-  readyToWhitelistListings: List<ListingWrapper>;
-  inChallengeCommitListings: List<ListingWrapper>;
-  inChallengeRevealListings: List<ListingWrapper>;
-  canBeUpdatedListings: List<ListingWrapper>;
```

Comment:
Looks like the `canBeUpdatedListings` phase -- which I understand to be 'can Resolve Challenge' got dropped in this refactor. 

Is this no longer a valid phase? Is it wrapped into `awaitingAppealRequestListings`?

Reply:
Nothing was ever populating that list, so i killed it